eng/coredistools.targets

```diff
@@ -0,0 +1,15 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.CoreDisTools" Version="$(MicrosoftNETCoreCoreDisToolsVersion)">
+      <ExcludeAssets>native</ExcludeAssets>
+      <GeneratePathProperty>true</GeneratePathProperty>
+    </PackageReference>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <CoreDisToolsLibrary Condition="'$(TargetOS)' == 'Windows'">$(PkgMicrosoft_NETCore_CoreDisTools)\runtimes\win-$(TargetArchitecture)\native\coredistools.dll</CoreDisToolsLibrary>
```

Comment:
@Anipik `TargetOS` for Windows should be lowercase `windows`, right?

Reply:
That is correct.

Reply:
Let me follow up on this

Reply:
Thanks. I saw this in another place in your PR as well. It isn't causing a problem right now as msbuild is case insensitive but it should be cleaned up.

Reply:
Should be fixed in https://github.com/dotnet/runtime/pull/44845

Reply:
What was the reason to have `Linux` and `OSX` uppercased but not `Windows`?